subsys/bluetooth/controller/ll_sw/ull_df_types.h

```diff
@@ -28,7 +28,15 @@ enum df_switch_sample_support {
  * - for ULL -> LL(HCI).
  */
 #if defined(CONFIG_BT_PER_ADV_SYNC_MAX) && defined(CONFIG_BT_CTLR_DF_PER_SCAN_CTE_NUM_MAX)
-#define IQ_REPORT_CNT (CONFIG_BT_PER_ADV_SYNC_MAX * CONFIG_BT_CTLR_DF_PER_SCAN_CTE_NUM_MAX * 2)
+#define SYNC_IQ_REPORT_CNT (CONFIG_BT_PER_ADV_SYNC_MAX * CONFIG_BT_CTLR_DF_PER_SCAN_CTE_NUM_MAX * 2)
 #else
-#define IQ_REPORT_CNT 0
+#define SYNC_IQ_REPORT_CNT 0
```

Comment:
Not sure what actually goes here, but should it have been `0U` instead? 

Ditto for `CONN_IQ_REPORT_CNT`

Reply:
Its value is in range `0-16`, so yes it should be `0U`.